syntax = "proto3";

message TofData {
  uint32 dist = 1;
}
